kuma/api/v1/serializers.py

```diff
@@ -10,8 +10,9 @@ class BCSignalSerializer(serializers.Serializer):
     slug = serializers.CharField(max_length=255)
     locale = serializers.CharField(max_length=7)
     explanation = serializers.CharField(
-        allow_blank=True,
-        max_length=1000
+        # Make sure these match the constants in bcd-signal.jsx
+        max_length=1000,
```

Comment:
Ideally, I'd like to have all these numbers in something like `settings/common.py` configurable by an env var. And then, this integer would be fed into the `bc-signals.jsx` code somehow so that if we one day decide to go from 1,000 to 1,500 we can just edit it in 1 place. Sadly the tooling to share constants and settings between Django and React code is tricky and the overhead not worth it in comparison to live with a little bit of repetition.